atuin-client/src/settings.rs

```diff
@@ -477,6 +519,7 @@ impl Settings {
             .set_default("key_path", key_path.to_str())?
             .set_default("session_path", session_path.to_str())?
             .set_default("dialect", "us")?
+            .set_default("timezone", "+0")?
```

Comment:
I know this would make it more of a breaking change, but thinking forwards; I think it would be better to default to `local`